### ISSUE TITLE
Improve Javadocs on `package-info.java` in SQS module.

### ISSUE THREAD

**Issue by maciejwalkowiak** [awspring/spring-cloud-aws#491]
Each `package-info.java` javadoc contains:

```java
/**
 * Auto-configuration for Amazon SQS (Simple Queue Service) integrations.
 */
```

Which is a result of copy & paste. Ideally, each package has a javadoc explaining what's inside the package.

**Comment by tomazfernandes** [comment]
Thanks for catching it, and sorry for the `CopyPasteException`! 😄

**Comment by maciejwalkowiak** [comment]
Fixed in #494
